docs/deployment-guide.md

```diff
@@ -59,28 +59,42 @@ If it's your first time with Ansible, please have a read at [Ansible concepts](h
 
 ## Getting started quickly with Vagrant
 
-The quickest way to get started and experiment with the playbook is by leveraging Vagrant to create a Virtualbox virtual machine to act as the control node **and** the target host.
+The quickest way to get started and experiment with the playbook is by
+leveraging Vagrant to create a Virtualbox virtual machine to act as the control
+node **and** the target host.
 
 1. Ensure your local machine has a minimum of 10G of memory and 4 CPUs
 2. Clone via Git or Download this repository to your local machine
 3. Install [Vagrant](https://www.vagrantup.com/downloads)
 4. Install [Virtualbox](https://www.virtualbox.org/wiki/Downloads)
 5. Open Virtualbox application to make sure it startup correctly
 6. In a terminal, navigate to where you cloned or unpacked the repository
-7. Set environment variables to hold your Nexus credentials as shown below (replacing the values appropriately):
+7. Set environment variables to hold your Nexus credentials as shown below
+   (replacing the values appropriately):
 
     ```bash
     export NEXUS_USERNAME="<your-username>"
     export NEXUS_PASSWORD="<your-password>"
     ```
 
-8. Run the main vagrant:
+8. Make sure to add the `known_urls` variables in the file `group_vars/repository.yml`.
+   It should contain any URL which is allowed to query the repository and the
+   first entry MUST be set to the dmain URL used to access Alfresco.
+   For example if your VM listens on 192.168.023:
+
+    ```yaml
+    known_urls:
+      - http://192.168.0.23/share
```

Comment:
in vagrant ip has been hardcoded to 192.168.56.100
https://github.com/Alfresco/alfresco-ansible-deployment/blob/master/Vagrantfile#L7